src/plugins/intel_gna/src/transformations/gather_sinking_matmul.cpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/gather_sinking_matmul.hpp"
+
+#include <transformations/utils/utils.hpp>
+#include <utility>
+
+#include "common/graph_utils.hpp"
+#include "openvino/cc/ngraph/itt.hpp"
+#include "openvino/opsets/opset12.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/rt_info/gather_sinking_attr.hpp"
+#include "transformations/utils/gather_sinking_utils.hpp"
+#include "transformations/utils/transformation_helper.hpp"
+
+using namespace ov;
+using namespace ov::opset12;
+using namespace ov::pass::pattern;
+using namespace ov::op::util;
+using namespace gather_sinking;
+using namespace ov::intel_gna::graph_utils;
+using namespace ov::intel_gna::pass;
```

Comment:
A question to all of the edited/added code: don't we want to use opset11?

Reply:
As I have been told by a ngraph::core team, we should currently use opset12